docs/dev/how-to/debug_permissions.mdx

```diff
@@ -0,0 +1,68 @@
+3. At the bottom of the **Permissions** page you'll find a list of repositories the user has access to and the reason for their access. Possible access reasons are:
+    - **Public** - The repository is marked as public on the code host, so all users on the Sourcegraph instance can access this repository.
+    - **Unrestricted** - The code host connection that this repository belongs to is not enforcing authorization, so all users on the Sourcegraph instance can access this repository.
+    - **Permissions Sync** - The user has access to this repository because they have connected an external account that has access to this repository on the code host.
+    - **Explicit API** - The user has been granted explicit access to this repository by means of the explicit permissions API.
+![](https://storage.googleapis.com/sourcegraph-assets/Docs/how-to/debug-permissions/accessible_repos.png)
+4. If the user should have access to a repository, but the repository is not at all present in this list, check if the user has connected their code host account to their user profile.
+
+## Check the user's Account security page
+
+1. As a site administrator, navigate to the user's settings and select **Account security**.
+![](https://storage.googleapis.com/sourcegraph-assets/Docs/how-to/debug-permissions/account_security_page.png)
+    - This page shows a user's connected external accounts. An external account is required to sync user permissions from code hosts.
+2. If a user has not connected their code host account, the user will have to do that by visiting their Account security settings and clicking on Add. Without a connected code host account, Sourcegraph is not able to associate their Sourcegraph identity with a code host identity to map permissions. If you don’t see the provider for your code host here, you have to configure it first.
+3. If the account is connected, you can try visiting the Permissions page again and schedule another permissions sync and see if anything changes. If nothing is changing when a permission sync completes, check the outbound requests log to verify that code host API calls are succeeding.
+
+## Check the outbound requests log
+
+1. On the Site admin settings page, click on the Outbound requests option near the bottom.
+![](https://storage.googleapis.com/sourcegraph-assets/Docs/how-to/debug-permissions/outbound_requests_option.png)
+2. If outbound requests aren’t enabled, you’ll need to enable it first. In the site configuration, add `"outboundRequestLogLimit": 50`, (50 indicates the number of most recent requests to store. Adjust as needed).
+3. The Outbound requests page will show a list of all the http requests Sourcegraph is making to outside services. Calls made to do user/repo permissions syncs will show up here as well. You can try to catch a user permission sync “in the act” by scheduling a sync, and then visiting the Outbound requests page.
```

Comment:
"`suggestion
3. The Outbound requests page will show a list of all the HTTP requests Sourcegraph is making to outside services. Calls made to do user/repo permissions syncs will also show up here. You can try to catch a user permission sync "in the act" by scheduling a sync and visiting the Outbound requests page.
```